plugins/feedback-backend/package.json

```diff
@@ -31,35 +31,35 @@
     "start": "backstage-cli package start",
     "build": "backstage-cli package build",
     "lint": "backstage-cli package lint",
-    "test": "backstage-cli package test",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
     "tsc": "tsc"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.19.8",
-    "@backstage/backend-plugin-api": "^0.6.8",
-    "@backstage/catalog-client": "^1.5.1",
+    "@backstage/backend-common": "^0.20.1",
+    "@backstage/backend-plugin-api": "^0.6.9",
+    "@backstage/catalog-client": "^1.5.2",
     "@backstage/catalog-model": "^1.4.3",
-    "@backstage/config": "^1.0.8",
+    "@backstage/config": "^1.1.1",
     "@types/express": "*",
     "axios": "^1.6.4",
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
-    "knex": "2.4.2",
+    "knex": "^3.1.0",
     "node-fetch": "^2.6.7",
     "nodemailer": "^6.9.8",
     "short-uuid": "^4.2.2",
     "winston": "^3.2.1",
     "yn": "^4.0.0"
   },
   "devDependencies": {
-    "@backstage/cli": "0.23.0",
-    "@types/nodemailer": "^6.4.14",
-    "@types/supertest": "^2.0.12",
-    "msw": "^1.0.0",
-    "supertest": "^6.2.4"
+    "@backstage/cli": "0.25.1",
+    "@types/nodemailer": "6.4.14",
+    "@types/supertest": "2.0.12",
+    "msw": "1.0.0",
```

Comment:
Hey can you update the msw package version to `1.3.2`, it is the reason failing tests.

Reply:
@yashoswalyo updated the package version.